engine/lib/access.php

```diff
@@ -63,15 +63,14 @@ function elgg_get_ignore_access() {
  * @see get_access_array()
  *
  * @param int  $user_guid User ID; defaults to currently logged in user
- * @param int  $site_guid Site ID; defaults to current site
  * @param bool $flush     If set to true, will refresh the access list from the
  *                        database rather than using this function's cache.
  *
  * @return string A list of access collections suitable for using in an SQL call
  * @access private
  */
-function get_access_list($user_guid = 0, $site_guid = 0, $flush = false) {
-	return _elgg_services()->accessCollections->getAccessList($user_guid, $site_guid, $flush);
+function get_access_list($user_guid = 0, $flush = false) {
+	return _elgg_services()->accessCollections->getAccessList($user_guid, $flush);
```

Comment:
You kept the second attribute in the access collections table.  This won't work.